### DISCUSSION TITLE
Good projects to land data analyst role

### DISCUSSION THREAD

**Discussion by Amarajj** [community/community#145743] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Can anyone suggest Good projects to land data analyst jobs?  I am new here and willing to collaborate with someone working on a project.

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @Amarajj!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!

**Reply by Amarajj** [reply]
Thank you
